tests/cypress/support/pages/carbon/RolesManagementPage.js

```diff
@@ -17,32 +17,31 @@
  * under the License.
  */
 class RolesManagementPage {
-    getUrl(){
+    getUrl() {
         return "/carbon/role/role-mgt.jsp";
     }
-    getRoleNameTextBox(){
+    getRoleNameTextBox() {
         return cy.get('input[name="org.wso2.carbon.role.filter"]');
 
     }
-    getSearchRolesButton(){
+    getSearchRolesButton() {
         return cy.get('input[value="Search Roles"]')
     }
 
-    getDeleteButtonOfRole(roleName){
-        return cy.get(`[onClick="deleteUserGroup(\\'${roleName}\\')"]`)
+    getDeleteButtonOfRole(roleName) {
+        return cy.get(`[onClick="deleteUserGroup(\\'${roleName.replace('/', '%2F')}\\')"]`)
```

Comment:
Let's check if there's a better way like an id or test id

Reply:
Since this is a part of the carbon console, it will be kept as it is